examples/xgboost_cv.py

```diff
@@ -0,0 +1,86 @@
+"""
+Optuna example that performs Cross Validation for cancer dataset
+using XGBoost.
```

Comment:
[nit] Please add a blank line after the first sentence like other examples.
```suggestion
Optuna example that performs cross-validation for cancer dataset using XGBoost.

```